pyRserve/rparser.py

```diff
@@ -517,13 +517,13 @@ def _postprocessData(self, data):
                 # For convenience reasons type-convert it into a native
                 # Python data type:
                 data = data[0]
-                if isinstance(data, (numpy.float, numpy.float64)):
```

Comment:
What do you think about using `float`, `int`, and `complex` directly, instead of the numpy variants with a trailing underscore? 
At least this is what numpy suggests itself in its deprecation warning:
```
DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`.
```

Reply:
Using the plain types directly sounds like better code.  

I've cleaned up the deprecated typenames in this small section of code.